### DISCUSSION TITLE
Select a specific go version

### DISCUSSION THREAD

**Discussion by adrientiburce** [golangci/golangci-lint#2314] [Q&A]
Hello everyone,

Since `go:1.17` a transition has started to remove the `// +build` flag and use only `//go:build`.

I understood that during the 1.17 version, linters `gofmt` and `goimports` will add the second flag and keep the first one : as explained [here](https://golang.org/doc/go1.17#gofmt)

The thing is my team uses `go1.16.5` so we would like to still use `golangci-lint` and the `gofmt` linter locally without having every build tag being modified.

So **my question is** : is there a way to choose a specific version when running `golangci-lint` ?
Also I tried to disable this specific rule, but I didn't find a way of doing that neither.

Thanks for your help !

**Comment by Antonboom** [comment]
Hello!

According to https://golangci-lint.run/usage/configuration/, please try
```
  # Define the Go version limit.
  # Mainly related to generics support since go1.18.
  # Default: use Go version from the go.mod file, fallback on the env var `GOVERSION`, fallback on 1.18
  go: '1.19'
```
